### PR TITLE
Rchu

### PR DESCRIPTION
**Pull request by unusualpineapple**
richard being lazy and not wanting to submit his own change

### CODE REVIEW

File: flask_app/config/mysqlconnection.py [hunk 1]
```diff
@@ -7,12 +7,12 @@ class MySQLConnection:
...
         # change the user and password as needed
         connection = pymysql.connect(host='localhost',
-                                    user='root',
-                                    password='root',
-                                    db=db,
-                                    charset='utf8mb4',
-                                    cursorclass=pymysql.cursors.DictCursor,
-                                    autocommit=True)
+                                     user='root',
+                                     password='rootroot',
```
Comment (unusualpineapple) [inline flask_app/config/mysqlconnection.py:11]: root root

Comment (unusualpineapple) [reply flask_app/config/mysqlconnection.py:11]: looks good

**Review by unusualpineapple — COMMENTED** [review]
looks good rich lets do it

**Comment by rchuu** [comment]
this sux.

**Pull request closed without merging**